README.md

```diff
@@ -0,0 +1,17 @@
+The previous version of LLVM semantics is created by David and Chucky.
+It is located in https://github.com/kframework/llvm-semantics-old.
+
+Versions of tools used to generate the .ll files:
+qangel@qangel-Satellite-S855:~/project/llvm-semantics$ opt -version
```

Comment:
What the hack is `qangel@qangel-Satellite-S855:~/project/llvm-semantics` and why is that particular information (the user and machine names, as well as the path) important to show to your users?